src/platformdirs/windows.py

```diff
@@ -12,6 +13,11 @@
 if TYPE_CHECKING:
     from collections.abc import Callable
 
+with contextlib.suppress(ImportError):
```

Comment:
I don't think this kind of way of defining will make the type checker happy. I imagine you want try/catch, set to None and do if statements on imports.

Reply:
When I run `hatch run type:run` locally, it doesn’t give me any errors about `ctypes` or `winreg` potentially not being defined. What happens when you run `hatch run type:run` locally?